partners/arrow-markets.json

```diff
@@ -0,0 +1,61 @@
+{
+    "name": "Arrow Markets",
+    "description": "Arrow Markets democratizes options trading through a hybrid RFQ system with on-chain exchange and settlement.",
+    "imgUrl": "https://x.com/ArrowMarkets/photo",
+    "missions": [
+      {
+        "name": "Arrow Markets_",
+        "description": "Complete liquidity provision and purchasing option on Arrow Markets",
+        "imgUrl": "https://img5.pic.in.th/file/secure-sv1/mission1.jpg",
+        "tags": ["DeFi"],
+        "quests": [
+          {
+            "name": "Liquidity Provision",
+            "description": "Provide Liquidity on Trader Joe for the arrow market pool",
+            "category": "DeFi",
+            "contract_address": "0x7F91272ff1A0114743D2df95F5905F9613Fd92b3",
+            "rewards": [
+              {
+                "type": "points",
+                "value": 60
+              }
+            ],
+            "tasks": [
+              {
+                "name": "Provide Liquidity",
+                "type": "POOL",
+                "description": "Provide Liquidity for Arrow Token on Trader Joe",
+                "contract_address": "0xc3311F358AFaD52e13425d41053F6fdCaf0DE6dB"
```

Comment:
A method_id which is the id of the contract function for this task must be present

Reply:
could you refer to me what I should be adding for the following transaction? This is the add liquidity transaction
https://snowscan.xyz/tx/0x18b4468ce7443d4cedfa0eea2f3359e22aafa8418ef2e3782694626c1da1b5b7

Reply:
Checking the transaction, the mint function is called to add liquidity, and it has the method_id 0x383d15c5

Reply:
@grandmastr should we use this contract address then?
0x18556da13313f3532c54711497a8fedac273220e

based on the tx I sent you

Reply:
Yes, since that's what handles the transactions, you just need to include the method_id for the actions for easier tracking on our end. Thanks